config/samples/core_v1beta1_openstackcontrolplane.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: core.openstack.org/v1beta1
 kind: OpenStackControlPlane
 metadata:
-  name: openstack
+  name: openstack-basic
```

Comment:
One of the test failures is because there is a mismatch between the name of the `OpenstackControlPlane` here and in the assert file: https://github.com/openstack-k8s-operators/openstack-operator/blob/08801ea0fd19cfeebc2c6d6f9d520cc2e35397b6/tests/kuttl/common/assert-sample-deployment.yaml#L4